src/projects.h

```diff
@@ -226,12 +226,18 @@ typedef struct ARG_list {
 	/* base projection data structure */
 
 
+#ifdef PJ_LIB__
+    /* we need this forward declaration in order to be able to add a
+       pointer to struct opaque to the typedef struct PJconsts below */
+    struct opaque;
```

Comment:
@hobu: OK - I'll change the variable name from O to Q: I wanted something related to the word opaque, while also being in the same ballpark of the alphabet as the P used for the PJ object in most of the PJ_xxx.c files. But going from just-before-P to just-after-P, and from Opaque to opaQue should not make much of a difference (except for the intended of avoiding a potential for misreading).

I'll post a note about my intentions to the mailing list tomorrow.

Also, I'm right now working on adding rudimentary unit tests to all PJ_xxx.c files: I need this to make sure I do not make stupid blunders of the kind repaired in commits 83cdf16 and afebe65.

I suppose this will up the test coverage percentage significantly.